bundle/language_en.properties

```diff
@@ -1176,6 +1176,8 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = Parents are together
 angal.patient.patientbrowser.title                                                                     = Patient Browser
 angal.patient.patientphoto                                                                             = Patient Photo
+angal.patient.patientphoto.error.title                                                                 = Patient Photo Error
+angal.patient.patientphoto.error.filenotfound                                                          = Patient photo file not found
```

Comment:
As noted in the other PR, the current convention is to end new message keys with `.msg`.    We are slowing changing the non-compliant ones to the newer format.  In addition  sentence  formatting is preferred:  `Patient photo file was not found.`